### PR TITLE
Bubble chart

### PR DESCRIPTION
**Pull request by yvonnele**
- bubble-chart
- API for party data (i.e. GAL-TAN position and colors)

### CODE REVIEW

File: wdip-be/src/config/config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
     "ES_PASSWORD": null,
     "WDIP_MOTION_INDEX": "motions",
     "WDIP_MOTION_REQUEST_LOG_INDEX": "motion-request-log",
-    "WDIP_DEFAULT_PARTIES": ["s", "m", "l", "kd", "v", "sd", "c", "mp", "nyd"],
+    "WDIP_DEFAULT_PARTIES": ["s", "m", "l", "kd", "v", "sd", "c"],
```
Comment (markushont) [inline wdip-be/src/config/config.js]: Is this being used somewhere? Should probably keep at least 'mp'!

File: wdip-fe/src/modules/BubbleChart.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                return (prev.submitted < current.submitted) ? prev : current
+            }).submitted;
+
+            const maxSubmitted = this.props.results.reduce(function (prev: any, current: any) {
+                return (prev.submitted > current.submitted) ? prev : current
+            }).submitted;
+
+            this.normalizer = new Normalizer(minSubmitted, maxSubmitted);
+
+            const partyData = this.props.partyData;
```
Comment (markushont) [inline wdip-fe/src/modules/BubbleChart.tsx]: Can we move this further down to where it's being used and add some sanity checks like you did for props.results? What happens if we don't get a properly formatted partyData?

File: wdip-fe/src/modules/BubbleChart.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+            bubbles.datasets = this.props.results.map((result: any) => ({
```
Comment (markushont) [inline wdip-fe/src/modules/BubbleChart.tsx]: Maybe not necessary right now, but maybe we should validate the data coming in somehow and decide what to do if it isn't formatted the way we're expecting. We should handle the case where result.approved is undefined for example. Maybe we can do it outside of the component, in the function calling the api.

File: wdip-be/src/config/config.js (hunk 1 above)
Comment (yvonnele) [reply wdip-be/src/config/config.js]: My bad! Accidentally removed mp. It is used in BE, getMotionsByParty.js

File: wdip-fe/src/modules/BubbleChart.tsx (hunk 2 above)
Comment (yvonnele) [reply wdip-fe/src/modules/BubbleChart.tsx]: Good point! Will do

File: wdip-fe/src/modules/BubbleChart.tsx (hunk 3 above)
Comment (yvonnele) [reply wdip-fe/src/modules/BubbleChart.tsx]: Yes, that's true. I added a default value (0) which should be set if result.approved is undefined in the meantime.

**Pull request merged**